warehouse/templates/manage/project/publishing.html

```diff
@@ -115,9 +116,10 @@ <h3>GitHub</h3>
         </div>
       </div>
       <div>
-        <input type="submit" value="{% trans %}Add{% endtrans %}" class="button button--primary">
+        <input type="submit" value="{% trans %}Add{% endtrans %}" class="button button--primary" {{ "" if request.user.in_oidc_beta else "disabled" }}>
```

Comment:
```suggestion
        <input type="submit" value="{% trans %}Add{% endtrans %}" class="button button--primary">
```
Since we won't be rendering this if the user isn't in the beta...